resources/language/resource.language.es_es/strings.po

```diff
@@ -105,6 +123,10 @@ msgctxt "#30061"
 msgid "{0.title}\nAuthor: {0.author}\nVideos: {0.videoCount}"
 msgstr "{0.title}\nAutor: {0.author}\nVídeos: {0.videoCount}"
 
+msgctxt "#30062"
+msgid "Nueva búsqueda"
```

Comment:
"New search" missing? Is it a problem (I'm not too familiar with this localization system)?